workflow/src/main/kotlin/com/r3/corda/sdk/token/workflow/flows/MoveToken.kt

```diff
@@ -94,7 +101,8 @@ open class MoveTokenFungible<T: TokenType>(
         val amount: Amount<T>,
```

Comment:
we should all catch up and figure out what the optimal patterns for structuring issuance, move, redeem flows.